### PR TITLE
fix: saturated visited link color

### PR DESCRIPTION
**Pull request by starryeyez024**
## Component name

- all components

### Related issue

- https://github.com/patternfly/patternfly-elements/issues/1126

### Preview

https://deploy-preview-1127--happy-galileo-ea79c4.netlify.app/elements/pfe-card/demo/

### What has changed and why
 
- Update visited link color to something that would pass accessibility standards on saturated backgrounds

### Testing instructions

1. Go to pfe-card demo
2. Click on a link in the complement or accent card so that its "visited"
3. Notice the link color is no longer purple, but a light gray instead.
 

### Ready-for-merge Checklist

Check off items as they are completed.  Feel free to delete items if they are not applicable.

- [x] Expected files: all files in this pull request are related to one request or issue (no stragglers or scope-creep).
- [x] Tests have been updated to cover these changes.
- [x] Browser testing passed.
- [x] Repository compiles and tests pass.
- [x] Changelog updated (not needed for documentation updates).

### Merging

Please **squash** when merging and ensure your commit message uses [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) formatting.

### CODE REVIEW

File: elements/pfe-sass/maps/_colors.scss [hunk 1]
```diff
@@ -22,7 +22,7 @@ $pfe-colors: (
 
   text--on-saturated:          $pf-color-white,
   text--muted--on-saturated:   $pf-color-black-300,
-  link--on-saturated:          $pf-color-white,
+  link--on-saturated:          $pf-color-black-300,
```
Comment (castastrophe) [inline elements/pfe-sass/maps/_colors.scss:25]: You've changed the `link--on-saturated` to gray but I think you meant to change the `link--visited--on-saturated`

**Review by castastrophe — CHANGES_REQUESTED** [review]

**Review by castastrophe — APPROVED** [review]
+1 just needs changelog

**Comment by castastrophe** [comment]
Does this need any doc or test updates?

**Comment by starryeyez024** [comment]
browser testing results: https://docs.google.com/spreadsheets/d/1ET3MVUXzgR36CGkfU3wd4SCbj8Mc0eGUixQ5kxOmpsg/edit#gid=1660273076

**Comment by castastrophe** [comment]
> Does this need any doc or test updates?

Bump ^ @starryeyez024

**Review by kylebuch8 — APPROVED** [review]
Livery Going Through Machine (it was dirty)

**Review by castastrophe — APPROVED** [review]
👍 Pulled down and reviewed; updated baselines.

**Pull request merged**